pkg/expression/exprstatic/evalctx.go

```diff
@@ -465,6 +466,11 @@ func newSessionVarsWithSystemVariables(vars map[string]string) (*variable.Sessio
 			cs = []string{name, val}
 		case vardef.CollationConnection:
 			col = []string{name, val}
+		case vardef.TiDBRedactLog:
```

Comment:
why add this check？

Reply:
It became a global variable which can not be set by `SetSystemVar` or `set @@tidb_redact_log=xx`.